### PR TITLE
Pathfinder/Paizo - Minor data changes

### PR DESCRIPTION
**Pull request by vincent-lhote**
Ultimate Equipment - Added caster level bonus on shifter headband
Pathfinder Society Field Guide - added not full text description

### CODE REVIEW

File: data/pathfinder/paizo/roleplaying_game/ultimate_equipment/ue_equip_magic_items.lst [hunk 1]
```diff
@@ -768,15 +768,15 @@ Phylactery of Negative Channeling.MOD											TYPE:LesserMedium
...
+Shifter's Headband (WIS) +6													TYPE:Magic.Wondrous.Headband.LesserMajor																																COST:39000	WT:0																								SOURCEPAGE:p.253																																																																																																																																																																						BONUS:STAT|WIS|6|TYPE=Enhancement	BONUS:CASTERLEVEL|DESCRIPTOR.Polymorph|1
+Shifter's Headband (CHA) +6													TYPE:Magic.Wondrous.Headband.LesserMajor																																COST:39000	WT:0																								SOURCEPAGE:p.253																																																																																																																																																																						BONUS:STAT|CHA|6|TYPE=Enhancement	BONUS:CASTERLEVEL|DESCRIPTOR.Polymorph|1
```
Comment (LegacyKing) [inline data/pathfinder/paizo/roleplaying_game/ultimate_equipment/ue_equip_magic_items.lst]: Are there any spells with a Descriptor of Polymorph?

Comment (LegacyKing) [inline data/pathfinder/paizo/roleplaying_game/ultimate_equipment/ue_equip_magic_items.lst]: Only thing my quick search turns up is SUBSCHOOL.

**Comment by LegacyKing** [comment]
Post note: Travis failure is a previous orphan I fixed, that can be safely ignored.

**Comment by LegacyKing** [comment]
@vincent-lhote You going to address the DESCRIPTOR?

**Comment by vincent-lhote** [comment]
Sorry I missed the first comment on the issue.

**Pull request merged**